CMakeLists.txt

```diff
@@ -209,6 +209,18 @@ if (WITH_PRIMESIEVE)
     set(PKGS ${PKGS} "PRIMESIEVE")
 endif()
 
+set (WITH_FTRAPV no
+        CACHE BOOL "Build with Integer Overflow-Check Flag (-ftrapv)")
```

Comment:
For adding extra flags for compilation, there's no need of a CMake option. You can use CXXFLAGS environment variable. See in bin/test_travis.sh for how `-Werror` is added.